system-int-test/test/cmr/system_int_test/logging/runtime_change_test.clj

```diff
@@ -0,0 +1,119 @@
+(ns cmr.system-int-test.logging.runtime-change-test
+  "This tests the CMR runtime change logging capabilities"
+  (:require
+   [cheshire.core :as json]
+   [clojure.string :as string]
+   [clojure.test :refer :all]
+   [cmr.common.mime-types :as mt]
+   [cmr.common-app.services.logging-config :as common-logging-config]
+   [cmr.mock-echo.client.echo-util :as e]
+   [cmr.system-int-test.system :as s]
+   [cmr.system-int-test.utils.logging-util :as log]
+   [cmr.system-int-test.utils.ingest-util :as ingest]))
+
+(use-fixtures :each (ingest/reset-fixture
+                     {"provguid1" "PROV1"}
+                     {:grant-all-search? false
+                      :grant-all-ingest? false}))
+
+;bootstrap is different and virtual-product I had problems - will address these later.
```

Comment:
Use two semicolons and a space for comments. We mark these types of TODOs with a ticket number.

Reply:
Wrote tickets for these and added a TODO comment for each.  Also added a space